.github/workflows/build.yml

```
@@ -271,6 +271,37 @@ jobs:
             ${{ github.workspace }}/packages/**/*.tgz
             ${{ github.workspace }}/packages/serverless/dist-awslambda-layer/*.zip
 
+  job_browser_playwright_tests:
+    name: Browser Playwright Tests
+    needs: job_build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out current commit (${{ github.sha }})
+        uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '16'
+      - name: Check dependency cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.CACHED_DEPENDENCY_PATHS }}
+          key: ${{ needs.job_build.outputs.dependency_cache_key }}
+      - name: Check build cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.CACHED_BUILD_PATHS }}
+          key: ${{ env.BUILD_CACHE_KEY }}
+      - name: Run Playwright tests
+        env:
+          # Install browsers inside node_modules to cache
+          # https://playwright.dev/docs/ci#caching-browsers
+          PLAYWRIGHT_BROWSERS_PATH: 0
+        run: |
+          cd packages/integration-tests
+          yarn run install-browsers
```

The browsers are already installed by `job_build` when installing the dependencies for the `@sentry/browser` package, this would install the same browsers twice.

```suggestion
        run: |
          cd packages/integration-tests
```